nautobot/docs/user-guide/administration/installation/services.md

```diff
@@ -246,7 +296,7 @@ To update your custom tasks, you'll need to do the following.
 
 For example:
```

Comment:
We can probably either remove or at least default-collapse the entire "Migrating to Celery from RQ" section?

Reply:
Let's add a new issue for removing RQ. I agree. We have historical documents.